pysb/simulator/scipyode.py

```diff
@@ -636,9 +627,8 @@ def rhs(t, y, p):
     return rhs
 
 
-def _integrator_process(code_eqs, jac_eqs, num_species, num_odes, initials,
-                        tspan, param_values, integrator_name, compiler,
-                        integrator_opts, compiler_directives):
+def _integrator_process(initials, param_values, code_eqs, jac_eqs, num_species, num_odes, tspan,
+                        integrator_name, compiler, integrator_opts, compiler_directives):
```

Comment:
Is there a reason to change the argument order here, since you're calling this function using keyword arguments anyway?

Reply:
Yes. executor.map is calling the function using *args  ```fs = [self.submit(fn, *args) for args in zip(*iterables)]``` So we need to put the param_values and initials at the beginning of the _integrator_process function

Reply:
Got it. Since this is a private function and there's a rationale for the order change, I think it's fine.